tests/runtime/out_stackdriver.c

```diff
@@ -1591,6 +1621,88 @@ static void cb_check_timestamp_format_duo_fields_incorrect_type(void *ctx, int f
     flb_sds_destroy(res_data);
 }
 
+void flb_test_monitored_resource_common()
+{
+    int ret;
+    int size = sizeof(MONITORED_RESOURCE_COMMON_CASE) - 1;
+    flb_ctx_t *ctx;
+    int in_ffd;
+    int out_ffd;
+
+    /* Create context, flush every second (some checks omitted here) */
+    ctx = flb_create();
+    flb_service_set(ctx, "flush", "1", "grace", "1", NULL);
+
+    /* Lib input mode */
+    in_ffd = flb_input(ctx, (char *) "lib", NULL);
+    flb_input_set(ctx, in_ffd, "tag", "test", NULL);
+
+    /* Stackdriver output */
+    out_ffd = flb_output(ctx, (char *) "stackdriver", NULL);
+    flb_output_set(ctx, out_ffd,
+                   "match", "test",
+                   "google_service_credentials", SERVICE_CREDENTIALS,
+                   "resource", "gce_instance",
+                   NULL);
+
+    /* Enable test mode */
+    ret = flb_output_set_test(ctx, out_ffd, "formatter",
+                              cb_check_monitored_resource,
+                              NULL, NULL);
+
+    /* Start */
+    ret = flb_start(ctx);
+    TEST_CHECK(ret == 0);
+
+    /* Ingest data sample */
+    flb_lib_push(ctx, in_ffd, (char *) MONITORED_RESOURCE_COMMON_CASE, size);
+
+    sleep(2);
+    flb_stop(ctx);
+    flb_destroy(ctx);
+}
+
+void flb_test_monitored_resource_priority()
```

Comment:
Maybe I miss something, I didn't see any difference between this testcase and `flb_test_monitored_resource_common`, can you clarify the difference?

I'd like to see the labels in `logging.googleapis.com/monitored_resource` have higher priority than:
- `logging.googleapis.com/local_resource_id`
- the data retrieved from metadata server, if the labels can be retrieved from metadata server

Reply:
Added two cases for priority check. PTAL